typescript/01-tutorials/03-deployment/01-agentcore-deployment/prerequisites.yaml

```diff
@@ -0,0 +1,119 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
Great example overall! fyi, checkout this other approach where we provided the users with a script to create the resources https://strandsagents.com/latest/documentation/docs/user-guide/deploy/deploy_to_bedrock_agentcore/typescript/#option-1-using-a-script-recommended